src/Uno.UI.RuntimeTests/Tests/Windows_UI_Xaml_Controls/Given_Grid.cs

```diff
@@ -181,6 +183,48 @@ FrameworkElement GetChild(int gridCol, int gridRow, int? colSpan = null, int? ro
 			});
 		}
 
+#if __SKIA__
```

Comment:
Why `#if __SKIA__` is needed?

Reply:
injector.GetMouse()

Reply:
Let's make it `HAS_UNO` then. It matters in that if input injector becomes supported on other platforms, the test will start running on such platforms without modifying it.